stdlib/public/SwiftShims/MetadataSections.h

```diff
@@ -0,0 +1,63 @@
+//===--- MetadataSections.h -----------------------------------------------===//
```

Comment:
Are the other C headers also using C++-style (C99) comments? If yes, then this is fine.

Reply:
All the header files in the SwiftShims directory use  this comment style.

Reply:
Thanks for checking!